src/drivers/postgres.rs

```diff
@@ -56,3 +58,30 @@ impl Driver for Postgres {
         Ok(())
     }
 }
+
+fn mk_connection(url: &str) -> Result<Connection, MigrateError> {
+    let ctx = try!(SslContext::new(SslMethod::Sslv23));
```

Comment:
Aesthetically speaking a bit unfortunate, however it happens just once and is not a performance problem. I was thinking that it isn't worth complicating the code.